flavors/vpp/vpp_flavor.go

```diff
@@ -3,24 +3,24 @@ package vpp
 import (
 	"github.com/ligato/cn-infra/core"
 	"github.com/ligato/cn-infra/datasync"
-	//"github.com/ligato/cn-infra/datasync/kvdbsync"
 	"github.com/ligato/cn-infra/datasync/resync"
-	//"github.com/ligato/cn-infra/db/keyval/redis"
 	"github.com/ligato/cn-infra/flavors/etcdkafka"
 	"github.com/ligato/vpp-agent/plugins/defaultplugins"
 	"github.com/ligato/vpp-agent/plugins/govppmux"
 	"github.com/ligato/vpp-agent/plugins/linuxplugin"
+	"github.com/ligato/cn-infra/flavors/redis"
+	"github.com/ligato/cn-infra/flavors/rpc"
 )
 
 // Flavor glues together multiple plugins to translate ETCD configuration into VPP.
 type Flavor struct {
-	Base   etcdkafka.FlavorEtcdKafka
-	//Redis  redis.Plugin
-	//RedisSync kvdbsync.Plugin
-	Resync resync.Plugin
-	GoVPP  govppmux.GOVPPPlugin
-	Linux  linuxplugin.Plugin
-	VPP    defaultplugins.Plugin
+	EtcdKafka etcdkafka.FlavorEtcdKafka
```

Comment:
please use anonymous composition of pointers for flavors:
like this:
  *etcdkafka.FlavorEtcdKafka
  *redis.FlavorRedis
  *rpc.FlavorRPC